src/ddmd/astbase.d

```diff
@@ -1132,7 +1137,7 @@ struct ASTBase
         bool havetempdecl;
         TemplateInstance inst;
 
-        final extern (D) this(Loc loc, Identifier ident, Objects* tiards)
+        final extern (D) this(Loc loc, Identifier ident, Objects* tiargs)
```

Comment:
Oh the joy of sharing the internal name with the constructor name...

Reply:
yep, took 2 hours of debugging to get to this freakin' typo.